src/frontend/komponenter/ManuellJournalføring/ManuellJournalføring.tsx

```diff
@@ -19,8 +19,21 @@ import HentPerson from '../Felleskomponenter/HentPerson/HentPerson';
 import UIModalWrapper from '../Felleskomponenter/Modal/UIModalWrapper';
 import Skjemasteg from '../Felleskomponenter/Skjemasteg/Skjemasteg';
 import { KnyttTilBehandling } from './KnyttTilBehandling';
+import { Dokumenter } from './Dokumenter';
+import styled from 'styled-components';
 import { useAmplitude } from '../../utils/amplitude';
 
+const PageSplit = styled.div`
+    display: flex;
+    flex-direction: row;
+`;
+
+const Bakgrunn = styled.div`
```

Comment:
We are working on creating som guidelines on what is "OK" scaling. So it might difficult to know what would be correct here atm, but at least all the functionality should be reachable so that might be the best check for now, and then we can go through all the pages when we have the guidelines in place.